### DISCUSSION TITLE
Linking to a note outside the current folder results in error - am I missing any config?

### DISCUSSION THREAD

**Discussion by sukantamaikap** [epwalsh/obsidian.nvim#413] [Q&A]
Hi mate,

Thanks for this amazing plugin. I'm experiencing the below error:

`Ambiguous link to document <>`

This appears only when the linked note is in another subfolder. 

<img width="1417" alt="Screenshot 2024-02-19 at 8 52 06 pm" src="https://github.com/epwalsh/obsidian.nvim/assets/3388408/1bd5d381-05f3-4b90-8f3f-668b9f252bcd">

Am I missing any config?

[Here](https://github.com/sukantamaikap/my.lazyvim/blob/main/lua/plugins/nvim-obsidian.lua) is my config for reference.

**Comment by x0aa7i** [comment]
I have the same problem

edit: Disabling marksman lsp fixed the issue for me
